lte/gateway/python/magma/pipelined/app/vlan_egress_flow_msgs.py

```diff
@@ -0,0 +1,91 @@
+"""
+Copyright 2022 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+This file models common logic of the mandatory services ingress,
+middle and egress.
+"""
+
+from magma.pipelined.openflow import flows
+from magma.pipelined.openflow.magma_match import MagmaMatch
+from magma.pipelined.openflow.registers import TUN_PORT_REG, Direction
+
+
+def _get_vlan_egress_flow_msgs(
```

Comment:
```suggestion
def get_vlan_egress_flow_msgs(
```
This is used in another file, so it's not private.

Reply:
addressed with last push